requirements_dev.txt

```diff
@@ -11,5 +11,5 @@ numpy>=1.19.0
 
 pytest==6.2.4
 pyyaml>=5.4.1
-jsonschema
+jsonschema==3.2.0
```

Comment:
Would have been better as `<4`?

Reply:
How are you testing (how did you find out it was breaking?)

Reply:
https://github.com/Julian/jsonschema/issues/847